### ISSUE TITLE
how to package to .exe after obfuscate

### ISSUE THREAD

**Issue by saostad** [javascript-obfuscator/javascript-obfuscator#801]
Hi,
I am trying to package my code with [pkg](https://www.npmjs.com/package/pkg) after obfuscating my code but getting 
```
Dynamic require may fail at run time, because the requested file
  is unknown at compilation time and not included into executable.
  Use a string literal as an argument for 'require', or leave it
  as is and specify the resolved file name in 'scripts' option.
```
what's the best practice to create an output that be easy to create .exe file?

**Comment by sanex3339** [comment]
Hi. Right now you can use the `conditional comments` or `reservedStrings` to disable obfuscation of `require`.
But seems I have to create an option to ignore obfuscation of the strings inside `require` statements.

**Comment by saostad** [comment]
an option would be great because it has problem with almost all of my require imports.

**Comment by sanex3339** [comment]
I'll create the PR with this feature tomorrow

**Comment by sanex3339** [comment]
Released as `ignoreRequireImports` option in `2.8.0`

**Comment by saostad** [comment]
it is working now. thanks!
